Mathlib/Order/Defs/Unbundled.lean

```diff
@@ -186,7 +184,7 @@ section
 /-- `IsRefl` as a definition, suitable for use in proofs. -/
 def Reflexive := ∀ x, x ≺ x
 
-/-- `IsSymm` as a definition, suitable for use in proofs. -/
+/-- `Std.Symm` as a definition, suitable for use in proofs. -/
 def Symmetric := ∀ ⦃x y⦄, x ≺ y → y ≺ x
```

Comment:
Are there any plans to link `Std.Symm` with this? Or to just deprecate this predicate altogether?

Reply:
I vote for deprecating. The Zulip thread also refers to this, and I think Kim supports deprecating this as well.